### PR TITLE
Remove libkpod

### PR DESCRIPTION
**Pull request by mheon**
Finish removing the last remnant of libkpod, the CRI-O config. Replace getRuntime with a version that does not depend on the CRI-O config.

### CODE REVIEW

**Comment by mheon** [comment]
Travis needs another graph driver forced. I'll add a command line flag to set graph driver and add it to the travis build.

**Comment by rhatdan** [comment]
@bot, retest this please

**Comment by mheon** [comment]
bot, retest this please

**Comment by mheon** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
bot, retest this please

File: cmd/podman/utils.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ func getRuntime(c *cli.Context) (*libpod.Runtime, error) {
 	// TODO add option to parse configs (CRI-O configs?) to generate runtimes
```
Comment (rhatdan) [inline cmd/podman/utils.go]: Remove this comment. Perhaps remove all references to CRI-O.  I am fine with doing this as a separate PR>

**Comment by rhatdan** [comment]
LGTM, although I want to remove references to CRI-O from Code base.

File: cmd/podman/utils.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	"github.com/containers/storage"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+// Generate a new libpod runtime configured by command line options
+func getRuntime(c *cli.Context) (*libpod.Runtime, error) {
+	options := []libpod.RuntimeOption{}
+
+	// TODO add option to parse configs (CRI-O configs?) to generate runtimes
```
[the remark this answers is not in the document]
Comment (mheon) [reply cmd/podman/utils.go]: ACK. We still need a config file for libpod defaults, though. Should happen in a separate pull request though.

File: cmd/podman/utils.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+		}
+		if c.GlobalIsSet("storage-opt") {
+			storageOpts.GraphDriverOptions = c.GlobalStringSlice("storage-opt")
+		}
+
+		options = append(options, libpod.WithStorageConfig(storageOpts))
+	}
+
+	// TODO CLI flags for image config?
+	// TODO CLI flag for signature policy?
```
[the remark this answers is not in the document]
Comment (mheon) [reply cmd/podman/utils.go]: No, only realized they were missing when I wrote this. I can add cards for them.

File: cmd/podman/utils.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (rhatdan) [reply cmd/podman/utils.go]: I agree.   I would prefer we cat to the point that containers/image containers/storage and libpod are treated as low level libraries with their own defaults.  crio.conf will override the defaults if it so chooses. and buildah/podman have no config, you have to specify options if you don't want to use the library defaults.

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM, however golang 1.9 tests are failng:

```
$ make install.tools
touch "/home/travis/gopath/.gopathok"
if [ ! -x "/home/travis/gopath/bin/git-validation" ]; then \
		go get -u github.com/vbatts/git-validation; \
	fi
package golang.org/x/crypto/ssh/terminal: unrecognized import path "golang.org/x/crypto/ssh/terminal" (https fetch: Get https://golang.org/x/crypto/ssh/terminal?go-get=1: dial tcp 74.125.70.141:443: i/o timeout)
make: *** [.install.gitvalidation] Error 1
The command "make install.tools" failed and exited with 2 during .
```

**Comment by mheon** [comment]
@TomSweeneyRedHat I think that's a network-related flake. Unfortunately, we don't have an easy way to rekick Travis like we do with Papr

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@mheon if you go into the Details there is a Restart button on the right side that will restart failed tests.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by mheon** [comment]
@rhatdan Wow. How did I only learn this now?

**Comment by baude** [comment]
@rh-atomic-bot retry

**Comment by mheon** [comment]
@rh-atomic-bot retry

**Comment by mheon** [comment]
Homu having issues, merging manually

**Pull request merged**